_pages/dev/security/security-headers.md

```diff
@@ -0,0 +1,139 @@
+**TODO**: WIP, discussion with security team and ccv2 ongoing.
+
+HTTP security headers are a important part of website security. They protect your storefront against potential attacks, such as XSS, code injection, clickjacking, etc.
+
+This document describes various security headers that are in place or should be added by customers when they deploy a Spartacus based Storefront in production.
+
+[Owasp](https://owasp.org/www-project-secure-headers/) gives a good overview of the various headers that can be applied. We've used the headers outlined by Owasp and reflect the recommended headers for Spartacus.
+
+## HTTP Strict-Transport-Security
+
+The HSTS security header forces the web browser to access the storefront by https only. This contributes to protocol downgrading and cookie hijacking.
+
+This is a pretty basic header that should be applied by default to Spartacus applications.
+
+The following snippets shows an example header config:
+
+```
+Strict-Transport-Security: max-age=31536000 ; includeSubDomains
+```
+
+**TODO**
+
+- @Michael Verify if this is added in ccv2 by default (likely already on ingress controller)
+
+## HTTP Public Key Pinning (HPKP)
```

Comment:
Maybe we do not need to go into too much detail here. Just say that it is deprecated, and that Spartacus does not promote its use.